include/constants.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
```

Comment:
I think we have a defs file, that could be somewhat redundant. We can split type definition from constants maybe.

Reply:
Indeed I forgot about `ddprof_defs.h`, I could move the constants there once migrated to C++.